docs/guides/use_checkpointing.md

```diff
@@ -193,46 +221,52 @@ serialization.from_state_dict(custom_target, raw_state_dict)
     # ... Continue with your work...
 
 # ... Until a time when you want to wait until the save completes:
-am.wait_previous_save()  # Block until the checkpoint saving is completed.
+async_checkpointer.wait_until_finished()  # Blocks until the checkpoint saving is completed.
 checkpoints.restore_checkpoint(ckpt_dir, target=None, step=2)
 ```
 
 ## Multi-host/multi-process checkpointing
 
 JAX provides a few ways to scale up your code on multiple hosts at the same time. This usually happens when the number of devices (CPU/GPU/TPU) is so large that different devices are managed by different hosts (CPU). To get started on JAX in multi-process settings, check out [Using JAX in multi-host and multi-process environments](https://jax.readthedocs.io/en/latest/multi_process.html).
 
-In the [Single Program Multi Data (SPMD)](https://jax.readthedocs.io/en/latest/glossary.html#term-SPMD) paradigm with JAX [`pjit`](https://jax.readthedocs.io/en/latest/jax.experimental.pjit.html), a large multi-process array can have its data sharded across different devices (check out the `pjit` [JAX-101 tutorial](https://jax.readthedocs.io/en/latest/jax-101/08-pjit.html)). This kind of data array needs a special experimental JAX API—[`GlobalAsyncCheckpointManager`](https://github.com/google/jax/blob/c7dcd0913bc8b4878a7e45184553c331254b801a/jax/experimental/gda_serialization/serialization.py#L452)—to save and restore checkpoints. This API lets each host dump its data shards to a single shared storage, such as a Google Cloud bucket.
+In the [Single Program Multi Data (SPMD)](https://jax.readthedocs.io/en/latest/glossary.html#term-SPMD) paradigm with JAX [`pjit`](https://jax.readthedocs.io/en/latest/jax.experimental.pjit.html), a large multi-process array can have its data sharded across different devices (check out the `pjit` [JAX-101 tutorial](https://jax.readthedocs.io/en/latest/jax-101/08-pjit.html)). This kind of data array needs a special experimental JAX API—[`gda_serialization`](https://github.com/google/jax/blob/c7dcd0913bc8b4878a7e45184553c331254b801a/jax/experimental/gda_serialization/serialization.py)—to save and restore checkpoints. This API lets each host dump its data shards to a single shared storage, such as a Google Cloud bucket.
 
-Flax provides an easy interface for users to pass in a `GlobalAsyncCheckpointManager` and store pytrees with multi-process arrays in the same fashion as single-process pytrees. Just use [`flax.training.checkpoints.save_checkpoint_multiprocess`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#flax.training.checkpoints.save_checkpoint_multiprocess) with the same arguments.
+Orbax uses this API to support storing pytrees with multi-process arrays in the same fashion as single-process pytrees. However, we recommend using the asynchronized `orbax.AsyncCheckpointer` to save large multi-process arrays on another thread, so that you can do computation along side the saves. 
```

Comment:
Nit:

```suggestion
Orbax uses this API to support storing pytrees with multi-process arrays in the same fashion as single-process pytrees. However, it's recommended using the asynchronized `orbax.AsyncCheckpointer` to save large multi-process arrays on another thread, so that you can perform computation alongside the saves. 
```

(First person like "we" and "our" -> second person https://developers.google.com/style/person though "we" sounds OK, IMHO)